tests/utils/pytest_wrapper.py

```
@@ -11,14 +11,22 @@
+            except psutil.Error:
+                pass
 
-exit(code)
+    return code
+
+
+# The guard is required: "spawn" and "forkserver" child processes re-import
+# this module as __mp_main__, and Python 3.14 changed the default start
+# method on Linux from "fork" to "forkserver"
+if __name__ == "__main__":
+    exit(main())
```

Ruff (0.16.1)

[warning] Use `sys.exit()` instead of `exit`

Replace `exit` with `sys.exit()`

(PLR1722)